lib/cmd/common/build.js

```diff
@@ -195,6 +197,23 @@ function mutateGitRefArgs(args) {
   }
 }
 
+function addCredentialsParams(args, cb) {
+  //Fix missing creds names - RHMAP-9456
+  if (args.bundleId) {
+    common.findCredentialsByID(args.bundleId, function(err, data) {
+      if (!err) {
+        //If this is not set its just set as blank
+        if (data.bundleName) {
+          args.credentialName = data.bundleName;
+        }
+      }
+      return cb();
```

Comment:
@wei-lee I was thinking the same and was torn by this one. It's not a mandatory field so the build works with or without the credential name being passed. I wasn't sure if I should be blocking the build if this "label" call fails. More than happy to update if you think I should.